<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_view</class>
 <widget class="QWidget" name="login_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="4">
    <spacer name="button_spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="description_label">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QLineEdit" name="description_input">
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <spacer name="top_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="3">
    <widget class="QLineEdit" name="ammount_input">
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </item>
   <item row="15" column="3">
    <spacer name="bottom_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <spacer name="button_spacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="product_name_label">
     <property name="text">
      <string>Product Name</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLineEdit" name="product_name_input">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="21" column="4">
    <widget class="QPushButton" name="create_product_button">
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <widget class="QPushButton" name="back_button">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLabel" name="ammount_label">
     <property name="text">
      <string>Ammount</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QLabel" name="price_label">
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QLabel" name="size_label">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QLineEdit" name="price_input">
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QLineEdit" name="size_input">
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
